editor/editor_file_system.h

```diff
@@ -202,11 +215,10 @@ class EditorFileSystem : public Node {
 	HashSet<String> dep_update_list;
 
 	struct ScanProgress {
-		float low = 0;
-		float hi = 0;
+		mutable float hi = 0;
+		mutable int current = 0;
 		mutable EditorProgressBG *progress = nullptr;
```

Comment:
I wonder why it was designed like this. `increment()` is `const`, but it's not used in `const` context from what I see, so it's an inherently mutating method made constant for no reason.

Reply:
I don't known, I found it weird too, I just went with what it was already there. I think it's to be able to pass `ScanProgress` as a const pointer in methods.

Reply:
Yeah but the pointer has no reason to be const if it's being changed 🤔

Reply:
Done!